shell.nix

```diff
@@ -39,8 +39,8 @@ pkgs.stdenv.mkDerivation {
     z3
   ];
   shellHook = ''
-    export PATH=$PWD/node_modules/.bin/:$PWD/bin:$PATH
-    export KLAB_EVMS_PATH="''${KLAB_EVMS_PATH:-''${PWD}/evm-semantics}"
+    export PATH=${toString ./node_modules/.bin}:${toString ./bin}:$PATH
```

Comment:
Haha I had started working on this PR and then got distracted trying to use a nix function to join strings by `:`, but I don't think that's really necessary.